packages/flutter/lib/src/material/input.dart

```diff
@@ -16,6 +16,125 @@ import 'theme.dart';
 
 export 'package:flutter/services.dart' show TextInputType;
 
+class InputField extends StatefulWidget {
```

Comment:
I'd like to do a rename pass as well.  I think what we call Input could be called TextField, as it is in the Material spec.  FormField could be FormTextField. What I've called InputField is a little harder, maybe TextFieldInput. Perhaps we should find a way to not expose RawInput at all.